.github/workflows/create-design-review-issues.yml

```diff
@@ -0,0 +1,36 @@
+# Executes `gulp sweep-experiments` on a schedule.
+# If experiments are swept, a PR is created.
+
+name: Create Design Review Issues
+
+on:
+  schedule:
+    # First day of the month at 00:00:00
+    - cron: '0 0 1 * *'
+
+jobs:
+  sweep-experiments:
+    name: Create Design Review Issues
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set Up Node
+        uses: actions/setup-node@v2.1.5
+
+      - name: Generate
+        id: generate
+        run: |
+          issue=$(wget -q -O - https://raw.githubusercontent.com/ampproject/amphtml/master/build-system/tasks/create-design-review-issues/index.js | node)
```

Comment:
Felt silly to checkout the repo / npm install for a standalone script, so we can just download it.

Reply:
Clever.